examples/analysis/geospatial/proximity_metric.py

```diff
@@ -0,0 +1,51 @@
+"""
+Proximity to raster or vector
+=============================
+
+This example demonstrates the calculation of proximity distances to a raster or vector using :func:`~geoutils.Raster.proximity`.
+"""
+# %%
+# We open an example raster, and a vector for which we select a single feature
+
+# sphinx_gallery_thumbnail_number = 2
+import geoutils as gu
+
+rast = gu.Raster(gu.examples.get_path("everest_landsat_b4"))
+vect = gu.Vector(gu.examples.get_path("everest_rgi_outlines"))
+vect = gu.Vector(vect.ds[vect.ds["RGIId"] == "RGI60-15.10055"])
```

Comment:
Haha, it's a weakness of Vector still. We don't have a good way to do indexing without relying on the geopandas' dataset...

Reply:
We do, I just hadn't updated the examples :wink:. The `__getitem__` and `__setitem__` methods are simply passed to `.ds`!

Reply:
Cool !